optimum/modeling_base.py

```diff
@@ -157,6 +157,7 @@ def _from_pretrained(
         revision: Optional[Union[str, None]] = None,
         force_download: bool = True,
         cache_dir: Optional[str] = None,
+        provider: Optional[str] = None,
```

Comment:
Lets not add `provider` to the default `model_base` since it is specific to ORT. Lets go with `kwargs`

Reply:
done